Assets/Scripts/Control&Input/Peer2PeerTransport.cs

```diff
@@ -125,14 +125,15 @@ private void OnSpawnFPSPlayers(NetworkConnectionToClient connection, PlayerFPSIn
         var playerManager = player.GetComponent<PlayerManager>();
         bool isLocalClient = false;
         string playerName = "";
+        //TODO: Use this dictionary to control connection IDs and playerIdentities, instead of all this nonsene string.equals stuff
         if (SteamManager.Singleton.PlayerDictionary.TryGetValue(connection.connectionId, out var playerDetails))
         {
             playerManager.identity.playerName = playerDetails.Item1;
             isLocalClient = playerDetails.Item1.Equals(SteamManager.Singleton.UserName);
             playerName = playerDetails.Item1;
             playerManager.identity.color = PlayerInputManagerController.Singleton.PlayerColors[playerDetails.Item2];
         }
-        //NetworkServer.AddPlayerForConnection(connection, playerManager.gameObject);
+        NetworkServer.AddPlayerForConnection(connection, playerManager.gameObject);
```

Comment:
Question for later and not really feedback: Could this be moved 'till after other initialization stuff?